src/shared/modules/config/wizard/v10/mapV10ConfigToV11.ts

```diff
@@ -9,7 +9,7 @@ import {ChartsConfigVersion, IndicatorTitleMode} from '../../../../types';
 export const mapV10ConfigToV11 = (config: V10ChartsConfig): V11ChartsConfig => {
     let extraSettings = config.extraSettings as V11CommonSharedExtraSettings;
     if (config.visualization?.id === WizardVisualizationId.Metric) {
-        extraSettings = {...config.extraSettings};
+        extraSettings = {...config.extraSettings, stacking: 'on'};
```

Comment:
Why was it necessary to add this default value for `WizardVisualizationId.Metric`?

Reply:
I've removed it